lego/apps/meetings/search_indexes.py

```diff
@@ -10,13 +10,12 @@ class MeetingModelIndex(SearchIndex):
     result_fields = (
```

Comment:
Have you tried changing the `queryset` to something along the lines of `Meeting.objects.filter(invited__includes=user.id)` (or some other variation that might be more correct:p)?